monitoring/rid_qualifier/flight_data_generator.py

```diff
@@ -332,8 +359,11 @@ def generate_rid_state(self, duration=180):
             rid_aircraft_flight_deserialized = self.make_json_compatible(rid_aircraft_flight)
             telemetery_data_list.append(rid_aircraft_flight_deserialized)
                     
+   
+        flight_operator_details = self.generate_flight_operator_details()
         
-        self.flight_telemetry = {"telemetery_data_list": telemetery_data_list, "reference_time": now_isoformat}
+        self.flight_telemetry = {"telemetery_data_list": telemetery_data_list, "reference_time": now_isoformat,'operator_details':flight_operator_details['operator_details'], 'flight_details':flight_operator_details['flight_details']}
```

Comment:
This value doesn't seem to match [the type declared for `self.flight_telemetry`](https://github.com/openskies-sh/dss/blob/6239634d779aeabd8869fe7cc848a4f5a917e065/monitoring/rid_qualifier/flight_data_generator.py#L111)

Reply:
Thanks! I have now declared the type of flight telemetry.